source/blog/2019-01-11-the-ember-times-issue-80.md

```diff
@@ -12,8 +12,10 @@ Grüezi mitenand Emberistas! 🐹
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [DecEmber is over? Collect your gift!](https://airtable.com/shrm9o5W89wsAa1Tq)
+[DecEmber](https://www.emberjs.com/blog/2018/11/29/december-event.html) has wrapped up and it seems to have bit a big hit! There were many contributions to `ember-learning` repos and it is all thanks to the awesome Ember community! 
```

Comment:
I think it would be:

```suggestion
[DecEmber](https://www.emberjs.com/blog/2018/11/29/december-event.html) has wrapped up and it seems to have been a big hit! There were many contributions to `ember-learning` repos and it is all thanks to the awesome Ember community! 
```